enterprise/backend/src/metabase_enterprise/serialization/api.clj

```diff
@@ -106,9 +117,14 @@
                                                     {:additive *additive-logging*})]
                    (try                 ; try/catch inside logging to log errors
                      (log/infof "Serdes import, size %s" size)
-                     (let [path (u.compress/untgz file dst)]
+                     (let [cnt  (u.compress/untgz file dst)
```

Comment:
Given that the return value of `untgz` has changed, I think its worth discussing the return value in its docstring.